data/json/monsters/mammal.json

```diff
@@ -1102,6 +1102,7 @@
     "vision_night": 4,
     "harvest": "dog_small_with_skull",
     "upgrades": { "age_grow": 42, "into": "mon_dog" },
+    "fear_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
```

Comment:
This is cool but they still copy the anger triggers from their parent, so they now have the same triggers for fear and anger. You're gonna have to delete the anger triggers from them as well.